js_modules/dagster-ui/packages/ui-core/worker-typings.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * A list of dependencies that are disallowed in WebWorkers.
+ */
+
+declare module '@vx/shape';
```

Comment:
We have this in the regular `typings.d.ts` as well. Not sure if there are types out in the wild for it that we could use instead.